.github/workflows/merge-release-tag.yml

```diff
@@ -0,0 +1,32 @@
+name: Merge Release Tag
+
+on:
+  push:
+    tags:
+      - 'mongosh@[0-9]+.[0-9]+.[0-9]+*'
```

Comment:
```suggestion
      - 'mongosh@[0-9]+.[0-9]+.[0-9]+'
```

I assume this is a regexp, in which case `+*` would be equivalent to just `*`

Reply:
Oh, and, as Nikola suggested above, shouldn't it be

```suggestion
      - 'mongosh@[0-9]+\.[0-9]+\.[0-9]+'
```

then?

Reply:
It's GitHub regex-esque pattern: https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
I think the idea is to allow `+rc0` though if it's just this we can also do
```yaml
      - 'mongosh@[0-9]+.[0-9]+.[0-9]+'
      - 'mongosh@[0-9]+.[0-9]+.[0-9]-rc[0-9]+'
```

Reply:
I am not sure how escapes would work but GitHub Docs themselves don't use them and use i.e `v[12].[0-9]+.[0-9]+`